lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -1476,22 +1488,62 @@ public void setTickInactiveTintList(@NonNull ColorStateList tickColor) {
   /**
    * Returns whether the tick marks are visible. Only used when the slider is in discrete mode.
    *
-   * @see #setTickVisible(boolean)
    * @attr ref com.google.android.material.R.styleable#Slider_tickVisible
+   * @deprecated Use {@link BaseSlider#getTickVisibilityMode()} to get the current
```

Comment:
Remove "BaseSlider". (I'm not sure if this will cause a problem because BaseSlider is a package private class. No harm to remove it.)